plan/preprocess_test.go

```diff
@@ -75,9 +75,8 @@ func (s *testValidatorSuite) TestValidator(c *C) {
 			errors.New("[schema:1068]Multiple primary key defined")},
 		{"create table t(c1 int not null, c2 int not null, primary key(c1), primary key(c2))", true,
 			errors.New("[schema:1068]Multiple primary key defined")},
-		{"alter table t auto_increment=1", true, errors.New("[autoid:3]No support for setting auto_increment using alter_table")},
-		{"alter table t add column c int auto_increment key, auto_increment=10", true,
-			errors.New("[autoid:3]No support for setting auto_increment using alter_table")},
+		{"alter table t auto_increment=1", true, nil},
+		{"alter table t add column c int auto_increment key, auto_increment=10", true, nil},
```

Comment:
This SQL is different with SQL in 78. Should we add tests in ddl_db_test.go?

Reply:
Done! PTAL @zimulala